commons-statistics-distribution/src/test/java/org/apache/commons/statistics/distribution/TruncatedNormalDistributionTest.java

```diff
@@ -200,7 +200,7 @@ void testVariance() {
     /**
      * Test cases for computation of the moments. This hits edge cases including truncations
      * too extreme to have a probability range for the distribution.
-     * The test ensures that the moments are computable for parameterisations
+     * The test ensures that the moments are computable for parameters-provided
```

Comment:
Parameterisations is the intended term. It is a British alternative spelling of [parametrization](https://en.wiktionary.org/wiki/parametrization). It is intended to describe the result of parameters, i.e. 'computable for the result of parameters (the distribution under test) where the bounds fall within'. Feel free to update it to the US english which is more standard for commons components.

Reply:
reverted